### PR TITLE
RF: _run_annex_* is deprecated in datalad core

### LINKED ISSUE #58: tests started to fail against datalad master
see/access logs at https://github.com/datalad/datalad-extensions/ . ok against maint

### PR DESCRIPTION
**Pull request by bpoldrack**
Remove calls that are deprecated in datalad core.
Sits on top of PR #59

Closes #58 (hopefully)

### CODE REVIEW

**Comment by bpoldrack** [comment]
Not clear to me what's going wrong on Travis, but considering this: https://travis-ci.com/github/datalad/datalad-ukbiobank/jobs/473060520#L571 should we also remove support for python 3.5, @mih ?

**Comment by mih** [comment]
fine with me

**Comment by bpoldrack** [comment]
FTR: Not forgotten - waiting to finish #59 first.

File: setup.cfg [hunk 1]
```diff
@@ -14,7 +14,7 @@ classifiers =
 [options]
 python_requires = >= 3.5
 install_requires =
-    datalad >= 0.13rc2
+    datalad >= 0.13.7
```
Comment (mih) [inline setup.cfg:17]:
```suggestion
    # really is a dependency on the to-be-released 0.14
    datalad >= 0.13.7
```

**Review by mih — APPROVED** [review]
Thx. LGTM! I have included a suggestion to annotate the true nature of the dependency declaration on datalad.

**Pull request merged**